### DISCUSSION TITLE
Welcome to md2enex Discussions!

### DISCUSSION THREAD

**Discussion by karloskalcium** [karloskalcium/md2enex#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by sleekspeed22** [comment]
Thanks for building this tool.  I have a question before using... Does it combine a folder of MD files into one Enex file for a notebook?  The reason I ask is I'm trying to migrate from Joplin where I have a number of notebooks (see subs set) and ultimately I want to move to Onenote...  So it looks like I need to export each note book -> series of MD files in folders -> ENEX -> import to Onenote...  
![image](https://github.com/karloskalcium/md2enex/assets/78045979/3c9f7298-76ec-4681-94dd-9ad9fe8e374f)

So I was hoping the notebook structure will be maintained in the Enex file.

**Reply by karloskalcium** [reply]
Yes - all files in the provided folder will be placed into a single `.enex` file. Note that it isn't currently recursive, so it will only read `.md` files in the top level of the provided directory.
